processing/src/main/java/org/apache/druid/query/QueryContexts.java

```diff
@@ -59,7 +59,7 @@
   public static final boolean DEFAULT_USE_CACHE = true;
   public static final boolean DEFAULT_POPULATE_RESULTLEVEL_CACHE = true;
   public static final boolean DEFAULT_USE_RESULTLEVEL_CACHE = true;
-  public static final Vectorize DEFAULT_VECTORIZE = Vectorize.FALSE;
+  public static final Vectorize DEFAULT_VECTORIZE = Vectorize.TRUE;
```

Comment:
Setting this to TRUE to see if any other tests fail.

Reply:
Reverting to false since TimeseriesQueryRunnerTest.testTimeseriesWithTimestampResultFieldContextForArrayResponse is failing. Will look at it seperately.